.github/workflows/perf_micro.yml

```diff
@@ -78,6 +78,17 @@ jobs:
         run: sh -x ./perf/tools/setup_env.sh
       - name: test
         run: make -f .test.mk test-perf
+        env:
+          # The taskset alone will pin all the process threads
+          # into a single (random) isolated CPU, see
+          # https://bugzilla.kernel.org/show_bug.cgi?id=116701.
+          # The workaround is using realtime scheduler for the
+          # isolated task using chrt, e. g.:
+          # sudo taskset 0xef chrt 50.
+          # But this makes the process use non-standard, real-time
+          # round-robin scheduling mechanism.
+          BENCH_CMD: "sudo taskset 0xfe chrt 50"
```

Comment:
Do we need `sudo` here since we ran the previous step as root?

Reply:
seems works fine without `sudo`, removed